File: Iniciante/URI 1035.ml
let a, b, c, d = Scanf.scanf "%i %i %i %i" (fun a b c d -> a, b, c, d);;

if (b > c && d > a && c > 0 && d > 0 && (c + d) > (a + b) && a mod 2 == 0) then (
    Printf.printf "Valores aceitos\n";
) else ( 
    Printf.printf "Valores nao aceitos\n";
)

File: Iniciante/URI 1035.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def A (Integer/parseInt (get line 0)))
(def B (Integer/parseInt (get line 1)))
(def C (Integer/parseInt (get line 2)))
(def D (Integer/parseInt (get line 3)))

(cond
    (and (> B C) (> D A) (> (+ C D) (+ A B)) (> C 0) (> D 0) (== 0 (mod A 2))) 
        (println "Valores aceitos")

    :else 
        (println "Valores nao aceitos")   
)